### PR TITLE
Redesigned UI

### PR DESCRIPTION
**Pull request by jonathantjendana**
image
image

### CODE REVIEW

File: src/main/java/seedu/moneygowhere/ui/GraphPanel.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+import javafx.scene.layout.Region;
+import javafx.scene.layout.StackPane;
+import seedu.moneygowhere.commons.core.LogsCenter;
+
+/**
+ * Tab containing the spending graph.
+ */
+public class GraphPanel extends UiPart<Region> {
+    private static final String FXML = "PlaceholderPanel.fxml";
+    private final Logger logger = LogsCenter.getLogger(GraphPanel.class);
```
Comment (choongyx) [inline src/main/java/seedu/moneygowhere/ui/GraphPanel.java]: Can remove the loggers if unused in order to pass Codacy test.

**Review by choongyx — APPROVED** [review]
Looks good! 👍👍👍

Can try deleting some of the unused files e.g statsChart and graphChart fxml and java files to increase coverage.

**Comment by minpyaemoe** [comment]
image

The black color pane is intended, right?

**Review by minpyaemoe — APPROVED** [review]

**Review by minpyaemoe — COMMENTED** [review]
image

The black color extra pane under the items are intended, right?

**Comment by Nanosync** [comment]
This is a good step towards a final design! I like the separators, very creative. However, I hope that the balance bar can be changed later. The reminders also seem to take up a lot of space and can be tweaked.

I cloned the repository and found some bugs:
- The background of the list panel is incorrect
image

- The graph/stats panel items keep shifting when you click another tab pane and hit any key or focus on another element.
image
image

File: src/main/java/seedu/moneygowhere/ui/GraphPanel.java (hunk 1 above)
Comment (jonathantjendana) [reply src/main/java/seedu/moneygowhere/ui/GraphPanel.java]: Done!

**Review by Nanosync — APPROVED** [review]

**Pull request merged**